overlay/etc/lighttpd/conf-available/50-tklcp.conf

```diff
@@ -1,3 +1,4 @@
+server.modules += ( "mod_accesslog" )
```

Comment:
Good catch man. Although we should probably add this to common and/or all appliances too I suspect!? Thoughts?

Reply:
lighttpd already ships with a catch-all access log config, it just has to be enabled which should be a single line change in `common/conf/lighttpd`. i will prepare the PR

Reply:
https://github.com/turnkeylinux/common/pull/292